DQM/HcalTasks/plugins/QIE10Task.cc

```diff
@@ -0,0 +1,160 @@
+	_cLETDCvsADC.initialize(_name, "LETDCvsADC",
+		new quantity::ValueQuantity(quantity::fQIE10ADC_256),
+		new quantity::ValueQuantity(quantity::fQIE10TDC_64),
+		new quantity::ValueQuantity(quantity::fN, true));
+	_cTETDCvsADC.initialize(_name, "TETDCvsADC",
+		new quantity::ValueQuantity(quantity::fQIE10ADC_256),
+		new quantity::ValueQuantity(quantity::fQIE10TDC_64),
+		new quantity::ValueQuantity(quantity::fN, true));
+	_cLETDC.initialize(_name, "LETDC",
+		new quantity::ValueQuantity(quantity::fQIE10TDC_64),
+		new quantity::ValueQuantity(quantity::fN, true));
+	_cADC.initialize(_name, "ADC",
+		new quantity::ValueQuantity(quantity::fQIE10ADC_256),
+		new quantity::ValueQuantity(quantity::fN, true));
+	_cOccupancy_depth.initialize(_name, hashfunctions::fdepth,
+		new quantity::DetectorQuantity(quantity::fiphi),
+		new quantity::DetectorQuantity(quantity::fieta),
+		new quantity::ValueQuantity(quantity::fN, true));
+	for (unsigned int j=0; j<10; j++)
+	{
+		_cLETDCvsADC_EChannel[j].initialize(_name,
+			"LETDCvsADC", hashfunctions::fEChannel,
+			new quantity::ValueQuantity(quantity::fQIE10ADC_256),
+			new quantity::ValueQuantity(quantity::fQIE10TDC_64),
+			new quantity::ValueQuantity(quantity::fN, true));
+		_cTETDCvsADC_EChannel[j].initialize(_name,
+			"TETDCvsADC", hashfunctions::fEChannel,
+			new quantity::ValueQuantity(quantity::fQIE10ADC_256),
+			new quantity::ValueQuantity(quantity::fQIE10TDC_64),
+			new quantity::ValueQuantity(quantity::fN, true));
+		_cADC_EChannel[j].initialize(_name,
+			"ADC", hashfunctions::fEChannel,
+			new quantity::ValueQuantity(quantity::fQIE10ADC_256),
+			new quantity::ValueQuantity(quantity::fN, true));
+		_cLETDC_EChannel[j].initialize(_name,
+			"LETDC", hashfunctions::fEChannel,
+			new quantity::ValueQuantity(quantity::fQIE10TDC_64),
+			new quantity::ValueQuantity(quantity::fN, true));
+	}
+
+	_cShapeCut_EChannel.book(ib, _emap, _filter_C36, _subsystem);
+	_cShapeCut.book(ib, _subsystem);
+	_cLETDCvsADC.book(ib, _subsystem);
+	_cTETDCvsADC.book(ib, _subsystem);
+	_cLETDC.book(ib, _subsystem);
+	_cADC.book(ib, _subsystem);
+	for (unsigned int i=0; i<10; i++)
+	{
+		char aux[10];
+		sprintf(aux, "TS%d", i);
+		_cLETDCvsADC_EChannel[i].book(ib, _emap, _filter_C36, _subsystem, aux);
+		_cTETDCvsADC_EChannel[i].book(ib, _emap, _filter_C36, _subsystem, aux);
+		_cLETDC_EChannel[i].book(ib, _emap, _filter_C36, _subsystem, aux);
+		_cADC_EChannel[i].book(ib, _emap, _filter_C36, _subsystem, aux);
+	}
+
+	_ehashmap.initialize(_emap, electronicsmap::fD2EHashMap, _filter_C36);
+}
+
+/* virtual */ void QIE10Task::endLuminosityBlock(edm::LuminosityBlock const& lb,
+	edm::EventSetup const& es)
+{
+	
+	//	finish
+	DQTask::endLuminosityBlock(lb, es);
+}
+
+/* virtual */ void QIE10Task::_process(edm::Event const& e, 
+	edm::EventSetup const&)
+{
+	edm::Handle<QIE10DigiCollection> cqie10;
+	if (!e.getByToken(_tokQIE10, cqie10))
+		std::cout << "Collection isn't available" << std::endl;
```

Comment:
Please replace with LogWarning, LogError or cms::Exception.
Use of std::cout is forbidden!